### PR TITLE
Use /upgrades resource

### LINKED ISSUE #6: Improvement of 'upgrade-cluster' command
It would be useful to make 'upgrade-cluster' command output more verbous, at least it should include upgrade ID.

### PR DESCRIPTION
**Pull request by ikalnytskyi**
Since PR #39 [1] to Kostyor, there's a new RESTful upgrade resource
available at URI:

```
/upgrades
```

Thus, we need to provide a commands to interact with it. This commit
adds the following commands:

```
$ kostyor upgrade-list <cluster>
$ kostyor upgrade-show <upgrade>
$ kostyor upgrade-start <cluster>
$ kostyor upgrade-pause <cluster>
$ kostyor upgrade-continue <cluster>
$ kostyor upgrade-cancel <cluster>
$ kostyor upgrade-rollback <cluster>
```

TODO: We need to consider to pass <cluster> instead of <upgrade> to
      upgrade-show command in order to be consistent with rest of the
      commands.

The commit also fixes #6 by providing detailed information about upgrade
tasks.

[1] https://github.com/sc68cal/Kostyor/pull/39

### CODE REVIEW

**Review by jvarlamova — COMMENTED** [review]
Looks good to me.

File: kostyor_cli/main.py [hunk 1]
```diff
@@ -26,18 +26,34 @@ def _make_request_with_cluster_id(http_method, endpoint, cluster_id):
...
+        )
+
+        # There are two reasons why using session is a good idea here. The
+        # first one is, session uses a pool with keep-alive connections so
+        # any further requests to the same domain won't initiate a new
+        # connection. The second reason is, it's configurable so we can
+        # setup, for example, common headers here and do not pass them
+        # across the code.
+        self.request = requests.Session()
+        self.baseurl = 'http://{0}:{1}'.format(host, port)
```
Comment (sc68cal) [inline kostyor_cli/main.py:55]: This is a change that is:
- Not discussed in the commit message
- Is a significant change

I would prefer that this be broken up into a separate commit

**Review by sc68cal — CHANGES_REQUESTED** [review]

File: kostyor_cli/main.py (hunk 1 above)
Comment (ikalnytskyi) [reply kostyor_cli/main.py:55]: You mean `baseurl` or `requests.Session()`?

Comment (sc68cal) [reply kostyor_cli/main.py:55]: This whole chunk of code. It's good that you're introducing it, but it should really be its own commit

Comment (ikalnytskyi) [reply kostyor_cli/main.py:55]: Agree. Do you ok with it generally?

Comment (sc68cal) [reply kostyor_cli/main.py:55]: yeah I think the change is very good, I just want to see it as a separate commit

Comment (sc68cal) [reply kostyor_cli/main.py:55]: It can still be part of this pull request, I just want you to break it out into its own commit

Comment (ikalnytskyi) [reply kostyor_cli/main.py:55]: yeah, i got it. i totally agree. i'll fix it.

**Review by sc68cal — APPROVED** [review]

**Pull request merged**